simulation/src/node.ts

```diff
@@ -162,14 +243,28 @@ export class Node {
           )
         }
         break
+      case MessageType.NotifyGroupTimeout:
+        if (this.id === 415) console.log(`#${this.id} timed out notification ${this.node?.children.length}`)
+        if (!this.node?.children.length) {
+          // The node still hasn't received children
+          // This occurs because others members don't know either, the protocol failed
+          messages.push(new Message(MessageType.StopSimulator, this.localTime, this.localTime, this.id, this.id, { status: StopStatus.SimultaneousFailures, targetGroup: this.node }))
```

Comment:
That's quite a long line. You should set up a linter to avoid this to follow cozy guidelines: https://github.com/cozy/cozy-guidelines#generic-code-style